src/pages/library/Code.js

```diff
@@ -51,89 +50,97 @@ export const makeWorkflowCard = ({ method, onClick }) => {
-    backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundColor: 'white',
-    backgroundSize: 27,
-    width: 37, height: 37,
-    marginRight: 13,
-    ...style
-  }
-})
+const LogoTile = ({ logoFile, ...props }) => {
+  return div(_.merge({
+    style: {
+      flexShrink: 0,
+      backgroundImage: `url(${logoFile})`,
+      backgroundRepeat: 'no-repeat', backgroundPosition: 'center',
+      backgroundSize: 27,
+      width: 37, height: 37,
+      marginRight: 13
+    }
+  }, props))
+}
 
-export const dockstoreTile = () => div({ style: { display: 'flex' } }, [
-  logoTile({ logoFile: dockstoreLogo }),
-  div([
-    h(Link, { href: `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=wdl&searchMode=files` }, 'Dockstore'),
-    div(['Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows'])
+export const DockstoreTile = () => {
+  return div({ style: { display: 'flex' } }, [
+    h(LogoTile, { logoFile: dockstoreLogo, style: { backgroundColor: 'white' } }),
+    div([
+      h(Link, { href: `${getConfig().dockstoreUrlRoot}/search?_type=workflow&descriptorType=wdl&searchMode=files` }, 'Dockstore'),
+      div(['Browse WDL workflows in Dockstore, an open platform used by the GA4GH for sharing Docker-based workflows'])
+    ])
   ])
-])
+}
 
-export const fcMethodRepoTile = () => div({ style: { display: 'flex' } }, [
-  logoTile({ logoFile: broadSquare, style: { backgroundColor: undefined, backgroundSize: 37 } }),
-  div([
-    h(Link, { href: `${getConfig().firecloudUrlRoot}/?return=${returnParam()}#methods` }, 'Broad Methods Repository'),
-    div([`Use Broad workflows in ${getAppName()}. Share your own, or choose from > 700 public workflows`])
+export const MethodRepoTile = () => {
+  return div({ style: { display: 'flex' } }, [
+    h(LogoTile, { logoFile: broadSquare, style: { backgroundSize: 37 } }),
+    div([
+      h(Link, { href: `${getConfig().firecloudUrlRoot}/?return=${returnParam()}#methods` }, 'Broad Methods Repository'),
+      div([`Use Broad workflows in ${getAppName()}. Share your own, or choose from > 700 public workflows`])
+    ])
   ])
-])
-
-
-const Code = ajaxCaller(class Code extends Component {
-  constructor(props) {
-    super(props)
-    const { featuredList, methods } = StateHistory.get()
-
-    this.state = { featuredList, methods }
-  }
-
-  async componentDidMount() {
-    const { ajax: { Methods } } = this.props
+}
 
-    const [featuredList, methods] = await Promise.all([
-      fetch(`${getConfig().firecloudBucketRoot}/featured-methods.json`).then(res => res.json()),
-      Methods.list({ namespace: 'gatk' })
+const Code = () => {
+  const signal = useCancellation()
+  const stateHistory = StateHistory.get()
+  const [featuredList, setFeaturedList] = useState(stateHistory.featuredList)
+  const [methods, setMethods] = useState(stateHistory.methods)
+  const [loading, setLoading] = useState(false)
+  const loadData = _.flow(
```

Comment:
Why not have this declaration within the `useOnMount`, so it's clearer visually that it's only used there?

Reply:
It's kind of a toss up between communicating the scope of that code vs. communicating the intent of the `useOnMount`. I'll change it.